ci/install.sh

```diff
@@ -35,10 +35,8 @@ if [ ! -z "${CRICTL_VERSION}" ]; then
     mv bin/crictl bin/crictl-${CRICTL_VERSION}
   fi
   cp bin/crictl-${CRICTL_VERSION} bin/crictl
-  # kubeadm being run by minikube complains that it fails to find 'crictl' in
-  # the system path on Travic CI jobs when using K8s 1.11, even if I add it like
-  # below ahead of time...
-  sudo cp bin/crictl-${CRICTL_VERSION} /usr/bin
```

Comment:
OH! But I notice that i copy to `/usr/bin` but the filename will be `crictl-<some-version>` so perhaps that is why I failed, while you do it right and also does it fancier with a symlink.

NICE!